### ISSUE TITLE
Like/Repost don't POSSE ?

### ISSUE THREAD

**Issue by rikmendes** [kylewm/KnownReactions#10]
It's been a few weeks I wanted to report that on my instance of Known, even though KnownReactions is correctly installed, when I like or repost something it doesn't get POSSEd when I select Twitter/Facebook before publishing. 

I'm not sure where to look at logs on my VPS to give you more details on what actually goes on on the backend but if you point me to it I can collect the log if that can help understand what's going on here.

**Comment by kylewm** [comment]
They're definitely supposed to POSSE to Twitter -- not Facebook (its API doesn't let us do anything fun anymore).

This is the code that handles likes (shares are right below) https://github.com/idno/Twitter/blob/master/Main.php#L308

Specifically see if you see any of those logging messages
```
"Could not find a status to like"
"Successfully posted like to Twitter: "
"Bad JSON response when posting a like to Twitter: "
```

**Comment by egoexpress** [comment]
@rikmendes I can confirm that POSSEing likes and reposts with the current version of the KnownReactions plugin works as expected. Shouldnt be a plugin issue.

**Comment by rikmendes** [comment]
i updated the plugin, but still can't get it to POSSE : https://www.rmendes.net/2016/liked-4-takeaways-from-pew%C3%A2s-social-media-news-report looking if anything on the log.

**Comment by rikmendes** [comment]
Just tested Repost and the same : https://www.rmendes.net/2016/repost-of-chronos-media-01b941f138 POSSE not working at all. I wonder where or what can I do to get to the end of the problem..

**Comment by rikmendes** [comment]
Strange thing just happend : I picked up a tweet to repost : https://twitter.com/EUWatchers/status/743364541661089796

on my instance it doesn't seems to have POSSED (at least no syndications links below the post) but it actually provoked a Retweet from Known, is it how repost is supposed to work ? 
https://www.rmendes.net/2016/repost-of-donald-trump-belgium-is-a-beautiful-city

is repost a retweet ?

**Comment by rikmendes** [comment]
I disabled all plugins un-necessary to do some tests, 
so the only plugin I have activated are the core plugins + Facebook/Twitter 
and of course the Reactions plugin. and it works : https://www.rmendes.net/2016/repost-of-kyle-mahans-tweet-a0951cf92c#comments POSSE works, now I just have to find the culprit plugins. 
because from a few minutes ago to now this post (with more plugins enabled) couldn't POSSE https://www.rmendes.net/2016/repost-of-a-new-way-to-fund-open-source-projects#comments and now I can each time I choose to repost something.
